packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -153,10 +167,45 @@ abstract class Scrollable extends StatefulComponent {
   ScrollableState createState();
 }
 
-/// Contains the state for common scrolling behaviors.
+/// Contains the state for common scrolling widgets.
 ///
-/// Widgets that subclass [Scrollable] typically use state objects that subclass
-/// [ScrollableState].
+/// Widgets that subclass [Scrollable] typically use state objects
+/// that subclass [ScrollableState].
+///
+/// The main state of a ScrollableState is the [scrollOffset]. The
+/// units of the scroll offset are defined by the specific subclass.
+/// By default, the units are logical pixels.
+///
+/// The "scroll offset" is the logical description of the current
+/// scroll position. It is a double.
```

Comment:
This paragraph seems slightly redundant with the previous paragraph.  Maybe merge them somehow?